tests/extract.rs

```diff
@@ -2,20 +2,20 @@ extern crate zip;
 
 use std::fs;
 use std::io;
-use std::path::PathBuf;
 
 use zip::ZipArchive;
 
 // This tests extracting the contents of a zip file
 #[test]
+#[ignore]
```

Comment:
Let's create an issue for the file sanitation problem and link to it here so we don't forget why this test is being ignored.

Reply:
Here we go https://github.com/mvdnes/zip-rs/issues/181

Reply:
It doesn't seem appropriate to leave the disabled API in the release code, so I've just removed it for now. We can bring it back when we fix path sanitization in 0.6